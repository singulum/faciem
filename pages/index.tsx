import Head from 'next/head'

export const HomePage = (): JSX.Element => (
  <div className="container">
    <Head>
      <meta charSet="utf-8" />
      <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
      <title>Curo</title>
      <meta
        name="description"
        content="Curo is the management dashboard for the Suus headless content management system"
      />

      <link
        rel="apple-touch-icon"
        sizes="180x180"
        href="/apple-touch-icon.png"
      />
      <link
        rel="icon"
        type="image/png"
        sizes="32x32"
        href="/favicon-32x32.png"
      />
      <link
        rel="icon"
        type="image/png"
        sizes="16x16"
        href="/favicon-16x16.png"
      />
      <link rel="manifest" href="/site.webmanifest" />
      <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#46494a" />
      <meta name="apple-mobile-web-app-title" content="Curo" />
      <meta name="application-name" content="Curo" />
      <meta name="msapplication-TileColor" content="#ffc40d" />
      <meta name="theme-color" content="#ffffff" />
    </Head>

    <main>
      <h1>Hello, World!</h1>
    </main>
  </div>
)

export default HomePage
