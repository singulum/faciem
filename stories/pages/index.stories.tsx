import React from 'react'
// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
import { Story, Meta } from '@storybook/react/types-6-0'

import { HomePage } from '../../pages/index'

export default {
  title: 'Pages/Home',
  component: HomePage,
} as Meta

const Template: Story = (args) => <HomePage {...args} />

export const Default = Template.bind({})
Default.args = {}
