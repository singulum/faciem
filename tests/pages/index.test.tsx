import React from 'react'
import { render } from '../testUtils'
import { HomePage } from '../../pages/index'

describe('Homepage', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<HomePage />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
