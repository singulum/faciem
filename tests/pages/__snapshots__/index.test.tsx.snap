// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <main>
      <h1>
        Hello, World!
      </h1>
    </main>
  </div>
</DocumentFragment>
`;
